cmd/finch/virtual_machine_init.go

```diff
@@ -86,6 +86,9 @@ func (iva *initVMAction) run() error {
 		return err
 	}
 
+	// ignore error, this is to ensure that the disk is only mounted once
```

Comment:
might need `//nolint:errcheck` here (and line 90)

e: and a few other places in this PR.

Reply:
I think assigning it to `_` works; I don't see it in the linter output. Is there any reason why `//nolint:errcheck` would be better?